packages/xo-server-audit/docs/specification.md

```diff
@@ -0,0 +1,78 @@
+This is a specification document which stipulate how the Audit log plugin should work.
+
+### Function
+
+Audit log is a plugin which add to XO the ability to keep a trace of the API calls.
+
+### Log construction
+
+To construct the logs, the plugin should handle two events:
+
+- `xo:preCall`: emitted before the call of the method
+- `xo:postCall`: emitted after the call of the method
```

Comment:
API calls is only one of the reported things, please list the various things that will be recorded, it will guide your specification.